stories/Button.stories.tsx

```diff
@@ -5,9 +5,13 @@ export default {
   title: "Example/Button",
   component: Button,
   args: {
-    ...Button.defaultProps,
```

Comment:
question: the reason I added this was so that Storybook Controls table show the default props selected. Like in Outline story, outline switch should be True, result radio Neutral, size radio Normal.

Any reason to remove this?

Reply:
Great question! If the addon does respect the args, then, because `...Button.defaultProps` is added to the meta, every story will essentially have every single arg set, which means that the amount of variants will always be one for each story. I didn't consider the controls panel though, so that's a valid concern 🤔